google-api-client/src/test/java/com/google/api/client/googleapis/services/AbstractGoogleClientRequestTest.java

```diff
@@ -203,7 +204,7 @@ public void testUserAgentSuffix() throws Exception {
     // Specify an Application Name.
     String applicationName = "Test Application";
     transport.expectedUserAgent = applicationName + " "
```

Comment:
You can inline AbstractGoogleClientRequest.USER_AGENT_SUFFIX  and applicationName here and the test becomes a little stronger since it's no longer duplicating as much of the logic in the model code.

Reply:
That constant is public, so I'm not sure if that would possibly break users.

Reply:
Oh, misread the comment. I'll update that.

Reply:
Using a string literal instead of a constant tests the constant. Using the constant simply guarantees that any mistakes in the constant value will not be caught by the tests. Yes, I've seen constants with bugs in them. :-(

Reply:
Done.